conf/SystemML-config.xml.template

```diff
@@ -65,6 +65,9 @@
    
    <!-- if codegen.enabled, compile literals as constants: 1..heuristic, 2..always -->
    <codegen.literals>1</codegen.literals>
+   
+   <!-- enables native blas for matrix multiplication and convolution, experimental feature -->
```

Comment:
I would recommend this be named "systemml.native.blas". But it's your call here. Going forward I would like that all systemml specific properties be prepended with "systemml."  So that when we read the properties files from other projects we depend on, we don't have name clashes.